lib/sie/parser.rb

```diff
@@ -33,8 +37,14 @@ def parse(io)
 
     private
 
+    attr_reader :options
```

Comment:
Yep https://github.com/barsoom/attr_extras#why-not-use-private-attr_reader-foo-instead-of-attr_private-foo